Source/WebKit/Platform/cocoa/WebPrivacyHelpers.mm

```diff
@@ -702,9 +702,26 @@ void configureForAdvancedPrivacyProtections(NSURLSession *session)
     });
 }
 
+bool isKnownTrackerAddressOrDomain(StringView host)
+{
+    TrackerAddressLookupInfo::populateIfNeeded();
+    TrackerDomainLookupInfo::populateIfNeeded();
```

Comment:
Hm.. it's probably fine in practice, but since `populateIfNeeded()` is asynchronous, we won't necessarily have available data the first time this is called.